frontend/src/components/ScheduledEventsTableCard/ScheduledEventsTableCard.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import axios from 'axios';
+import { Card } from 'react-bootstrap';
+import { Dimmer, Loader } from 'semantic-ui-react';
+import { ScheduledEventsTable } from '../ScheduledEventsTable/ScheduledEventsTable';
+import { api, localForageKeys } from '../../utils';
+import * as localForage from 'localforage';
+import { PaginationRow } from '../PaginationRow';
+
+export const ScheduledEventsTableCard = () => {
+  const [scheduledEvents, setScheduledEvents] = React.useState({ data: [] });
+  const [loading, setLoading] = React.useState(true);
+  const [count, setCount] = React.useState(0);
+  const [pageCount, setPageCount] = React.useState(0);
+  const [currentPage, setCurrentPage] = React.useState(1);
+  const [selectedPage, setSelectedPage] = React.useState(0);
+  const basePath = api.routes.VIEW_SCHEDULED_EVENTS;
+  const limit = 5;
```

Comment:
As a constant, convention is uppercase naming... can even move it out of the function... less crowding